src/routes/blog/post/behind-the-pr-tales-from-the-open-source-world/+page.markdoc

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Behind the pull request: tales from the open source world
```

Comment:
I think if you want to be able to use `:` in your title you need to wrap it in quotes